src/Sarif/Writers/SarifLogger.cs

```diff
@@ -121,18 +121,19 @@ private static Uri Redact(Uri uri, IEnumerable<string> tokensToRedact)
             IEnumerable<string> invocationPropertiesToLog = null,
             string defaultFileEncoding = null)
             : this(new StreamWriter(new FileStream(outputFilePath, FileMode.Create, FileAccess.Write, FileShare.None)),
-                  loggingOptions,
-                  dataToInsert,
-                  tool,
-                  run,
-                  invocationTokensToRedact,
-                  invocationPropertiesToLog)
+                  loggingOptions: loggingOptions,
+                  dataToInsert: dataToInsert,
+                  tool: tool,
+                  run: run,
+                  analysisTargets: analysisTargets,
```

Comment:
>analysisTargets [](start = 18, length = 15)

failure to explicitly provide analysisTarget argument results in invocationsToRedact argument being passed as targets list. gap in unit tests. we need this fix asap to fix issues in checked in binskim test files. PENDING